src/Shaders/background.fragment.fx

```diff
@@ -6,6 +6,8 @@ precision highp float;
 
 #include<__decl__backgroundFragment>
 
+#define RECIPROCAL_PI2 0.15915494

```

Comment:
why this change?

Reply:
Because the Shader was failing to compile without this line

Reply:
Re-confirmed. Shader fails to compile in chrome without that addition. If it's not the correct way or place for it to be added then something else is amiss as it is needed.

Reply:
Can you pinpoint where it is used. The shaders work without it so far so it must have to do with something you added.

Reply:
It's used in the `reflectionFunction.fx` file which always gets included in the backgroundMaterial. If you specify a reflection of an equirectangular nature, then it will get enabled by an ifdef. This is not code I added, but simply trying to use an equirectangular background reflection.

As for where and why I placed it, I was simply following the pattern other shaders had used for its implementation. If you would prefer it included in another file, importing a list of constants from another file, or instead wrapping it behind an ifdef too I can do so.

Reply:
OK nope this is fine. I just wanted to understand :)